Listener/KernelTerminateListener.php

```diff
@@ -15,25 +15,19 @@
 
 use AppInsightsPHP\Client\Client;
 use Psr\Log\LoggerInterface;
-use Psr\SimpleCache\CacheInterface;
 use Symfony\Component\Console\ConsoleEvents;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\KernelEvents;
 
 final class KernelTerminateListener implements EventSubscriberInterface
 {
-    public const CACHE_CHANNEL_KEY = 'app_insights_php.failure_cache';
-    public const CACHE_CHANNEL_TTL_SEC = 86400; // 1 day
-
     private $telemetryClient;
-    private $logger;
-    private $failureCache;
+    private $fallbackLogger;
 
-    public function __construct(Client $telemetryClient, CacheInterface $failureCache = null, LoggerInterface $logger = null)
+    public function __construct(Client $telemetryClient, LoggerInterface $fallbackLogger = null)
```

Comment:
why do we need 2 fallback loggers? I thought you added one to the `Client` so Client should actually never throw exception